rhizome/host/lib/vm_setup.rb

```diff
@@ -421,18 +422,28 @@ def cloudinit(unix_user, public_keys, gua, nics, swap_size_bytes, boot_image)
     end.join("\n")
 
     raparams = nics.map { "ra-param=#{_1.tap}" }.join("\n")
-
+    interfaces = nics.map { "interface=#{_1.tap}" }.join("\n")
+    dnsmasq_address_ip6 = NetAddr::IPv6.parse("fd00:0b1c:100d:53::")
```

Comment:
I like how this looks, but wonder: is there any other use of all the bits in `fd00:0b1c:100d:53` that's non-zero?

I think we should probably keep it as-is, it's rather easy to read, and I can see the obvious problem with `::53` (in that the special base subnet of `fd00:0b1c:100d` gets crowded and services have no wiggle room), but, dare to dream I guess?

Reply:
Maybe later we will find another case but for now, I think this is good enough.